src/pages/Confirmation/Confirmation.js

```diff
@@ -18,10 +18,12 @@ function Confirmation(props) {
   const [handleWords, setWords] = useState(false);
   const [handleVocab, setVocab] = useState(false);
   const [needRowSpace, setNeedRowSpace] = useState(false);
+  // Phonics Lessons
+  const [typeCLessons] = useState(["C", "C1", "C2", "C3"]);
```

Comment:
make this a const in the utils folder (i think)! state variables are component-specific and usually meant to change

Reply:
honestly I might just remove these (there's 1 or 2 other places I do this) and just put C3 in whenever I'm adding or checking the Phonics lesson type. The only reason I provided multiple C lesson types is I wasn't sure if we would need to support multiple types of phonics lessons down the line. Although based off of Sumit's email, we'd probably be fine just using C3